Source/CarthageKit/Project.swift

```diff
@@ -47,6 +47,12 @@ public enum ProjectEvent {
 
 	/// Building an uncached project.
 	case buildingUncached(Dependency)
+
+	/// Launching a git task.
+	case launchingGitTask(Task)
+
+	/// Running a git task.
+	case runningGitTask(String)
```

Comment:
I'm not sure that we need these events. I don't think we want to report these to users.